pkg/accountmanager/service.go

```diff
@@ -0,0 +1,135 @@
+package accountmanager
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Optum/dce/pkg/account"
+	"github.com/Optum/dce/pkg/arn"
+	"github.com/Optum/dce/pkg/common"
+	"github.com/Optum/dce/pkg/errors"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/aws/aws-sdk-go/service/sts/stsiface"
+	validation "github.com/go-ozzo/ozzo-validation"
+)
+
+// ServiceConfig has specific static values for the service configuration
+type ServiceConfig struct {
+	AccountID                   string   `env:"ACCOUNT_ID" envDefault:"111111111111"`
+	S3BucketName                string   `env:"ARTIFACTS_BUCKET" envDefault:"DefaultArtifactBucket"`
+	S3PolicyKey                 string   `env:"PRINCIPAL_POLICY_S3_KEY" envDefault:"DefaultPrincipalPolicyS3Key"`
+	PrincipalIAMDenyTags        []string `env:"PRINCIPAL_IAM_DENY_TAGS" envDefault:"DefaultPrincipalIamDenyTags"`
+	PrincipalMaxSessionDuration int64    `env:"PRINCIPAL_MAX_SESSION_DURATION" envDefault:"3600"` // 3600 is the default minimum value
+	AllowedRegions              []string `env:"ALLOWED_REGIONS" envDefault:"us-east-1"`
+	TagEnvironment              string   `env:"TAG_ENVIRONMENT" envDefault:"DefaultTagEnvironment"`
+	TagContact                  string   `env:"TAG_CONTACT" envDefault:"DefaultTagContact"`
+	TagAppName                  string   `env:"TAG_APP_NAME" envDefault:"DefaultTagAppName"`
+	PrincipalRoleDescription    string   `env:"PRINCIPAL_ROLE_DESCRIPTION" envDefault:"Role for principal users of DCE"`
+	PrincipalPolicyDescription  string   `env:"PRINCIPAL_POLICY_DESCRIPTION" envDefault:"Policy for principal users of DCE"`
+	tags                        []*iam.Tag
+	assumeRolePolicy            string
+}
+
+// Service manages account resources
+type Service struct {
+	client   clienter
+	storager common.Storager
```

Comment:
why not through `storager` into the `clienter` structs -- seems like that's where AWS service clients are being held

Reply:
Good question.  I'm using the clienter as a way to keep access to some aws configs that we can share across clients.